sdk/redis/Azure.ResourceManager.Redis/CHANGELOG.md

```diff
@@ -3,12 +3,19 @@
 ## 1.1.0-beta.1 (Unreleased)
 
 ### Features Added
-
+- ExportRdbContent has a new parameter PreferredDataArchiveAuthMethod
+- ImportRdbContent hasa new paramter PreferredDataArchiveAuthMethod
+- RedisLinkedServerWithPropertyCreateOrUpdateContent has a new parameter GeoReplicatedPrimaryHostName
+- RedisLinkedServerWithPropertyCreateOrUpdateContent has a new parameter PrimaryHostName
+- RedisLinkedServerWithPropertyData has a new parameter GeoReplicatedPrimaryHostName
+- RedisLinkedServerWithPropertyData has a new parameter PrimaryHostName
 ### Breaking Changes
-
+- Deleting a linked server is now a long running operation
```

Comment:
If there are breaking changes, we need to bump the major version, e.g. `2.0.0-beta.1`. @ArthurMa1978  FYI